src/torch/6-bboxlib/bboxlib.lua

```diff
@@ -53,29 +56,15 @@ function toOutputCoordinates(x, y, reduction, xdelta, ydelta)
   return toOutputCoordinate(x), toOutputCoordinate(y)
 end
 
-function toOutputCoordinate(coord)
-  return math.max(math.floor((math.floor((math.floor((coord - 10) / 4) - 2) / 2) - 2) / 2) - 2 + 1 - 10, 1)
-end
-
-function toImageCoordinates(x, y)
-  return toImageCoordinate(x), toImageCoordinate(y)
-end
-
-function toImageCoordinate(coord)
+local function toImageCoordinate(coord)
   return ((((((coord - 1 + 2 + 10) * 2) + 2) * 2) + 2) * 4) + 10 + 7 - 224
 end
 
---  Equations:
---    Ow = (Iw - delta) / reduction --> delta = Iw - (Ow * reduction)
---    Ox = (Ix - delta) / reduction
---    Ix = (Ox * reduction) + delta
```

Comment:
why was that removed?
Even if it's not used, in this case it shouldn't be removed as it might be used later on.